app/uk/gov/hmrc/customs/managesubscription/services/dto/Email.scala

```diff
@@ -22,4 +22,5 @@ case class Email(to: List[String], templateId: String, parameters: Map[String, S
 
 object Email {
   implicit val emailWrites: Writes[Email] = Json.writes[Email]
```

Comment:
You can remove previous writes.

Format contains reads and writes so it's enough. Now it's like a duplication of writes